backend/apid/routers/extensions.go

```diff
@@ -16,7 +16,7 @@ type ExtensionsRouter struct {
 }
 
 // NewExtensionRouter instantiates new router for controlling extension resources
```

Comment:
This comment doesn't match the new func name 🤦‍♀️

Reply:
Tabarnak

Reply:
> Tabarnak

hahahahaha

![](https://user-images.githubusercontent.com/4105580/38321485-ce38c25a-3805-11e8-85dd-cd1451af0fe4.gif)